src/components/side-panel/panel-header.js

```diff
@@ -123,17 +122,10 @@ const StyledPanelDropdown = styled.div`
 export const PanelAction = ({item, onClick}) => (
   <StyledPanelAction className="side-panel__panel-header__action"
     data-tip data-for={`${item.id}-action`} onClick={onClick}>
+    <p style={{fontWeight: 'bold'}}>Share</p>
     <a target={item.blank ? '_blank' : ''} href={item.href}>
       <item.iconComponent height="20px" />
     </a>
-    <Tooltip
```

Comment:
Last thoughts here, the `PanelAction` component is a generic component, and it should support rendering tooltip based on whether the item has one. In one of my applications, I am passing an array of actionItems to PanelHeader with tooltips. This change will break those.

I think it makes sense to leave the rendering of the Tooltip in there and have it based on whether the item has tooltip property.
so if you can change this to
{item.tooltip ? <Tooltip {...} /> : null} it would be safer.

Reply:
Thanks for your comment! I add back tooltip and send a commit to fix this issue. 

Thanks again